File: 8kyu/name-shuffler.js
/*The problem
Write a function that returns a string in which firstname is swapped with last name.*/

// My Solution

function nameShuffler(str){
    return str.split(" ").reverse().join(" ");
    }

// other solutions
const nameShuffler = str => str.split(' ').reverse().join(' ');


//for good measure...
function nameSuffle(str){
    var arr = str.split(" ");
    return arr[1] + ' ' + arr[0]
  }

File: 8kyupy/name-shuffler.py
#The Problem

#Write a function that returns a string in which firstname is swapped with last name.

#My Solution
def name_shuffler(str_):
    return ' '.join(str_.split(' ')[::-1])

#other solutions...
def name_shuffler(str_):
    [first, last] = str_.split()
    return last + " " + first

#for good measure...
name_shuffler = lambda s: ' '.join(reversed(s.split()))